src/Microsoft.FeatureManagement/FeatureFilters/Cron/CronExpression.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using System;
+
+namespace Microsoft.FeatureManagement.FeatureFilters.Cron
+{
+    /// <summary>
+    /// The Crontab expression with Minute, Hour, Day of month, Month, Day of week fields.
+    /// </summary>
+    internal class CronExpression
+    {
+        private static readonly int NumberOfFields = 5;
+        private static readonly char[] WhitespaceDelimiters = {' ', '\n', '\t'};
+
+        private readonly CronField _minute;
+        private readonly CronField _hour;
+        private readonly CronField _dayOfMonth;
+        private readonly CronField _month;
+        private readonly CronField _dayOfWeek;
+
+        public CronExpression(CronField minute, CronField hour, CronField dayOfMonth, CronField month, CronField dayOfWeek)
+        {
+            _minute = minute;
+            _hour = hour;
+            _dayOfMonth = dayOfMonth;
+            _month = month;
+            _dayOfWeek = dayOfWeek;
+        }
+
+        /// <summary>
+        /// Check whether the given expression can be parsed by a CronExpression.
+        /// If the expression is invalid, an ArgumentException will be thrown.
+        /// </summary>
+        /// <param name="expression">The expression to parse.</param>
+        /// <returns>A parsed CronExpression.</returns>
+        public static CronExpression Parse(string expression)
+        {
+            if (expression == null)
+            {
+                throw new ArgumentNullException(nameof(expression));
+            }
+
+            string InvalidCronExpressionErrorMessage = $"The provided Cron expression: '{expression}' is invalid.";
+
+            string[] fields = expression.Split(WhitespaceDelimiters, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Please be consistent about new line usage. The line above is separated by a blank line. Below it is not. I suggest we stick with adding blank line between statements.